src/commands/volume/volume.command.ts

```diff
@@ -49,21 +49,23 @@ export class VolumeCommand {
 
     const volume = dto.volume / 100;
 
-    this.logger.debug(
-      `Calculated volume ${volume} from dto param ${dto.volume}`,
-    );
+    this.logger.debug(`Calculated volume ${volume} from dto param ${dto.volume}`);
 
+    // 🔊 Change active audio resource volume
     this.discordVoiceService.changeVolume(volume);
 
-    // Discord takes some time to react. Confirmation message should appear after actual change
+    // 💾 Persist the change globally
+    this.playbackService.setVolume(volume);
+
+    // Wait for the change to propagate before confirming
     await sleepAsync(1500);
 
     await interaction.editReply({
       embeds: [
         this.discordMessageService.buildMessage({
           title: `Successfully set volume to ${dto.volume.toFixed(0)}%`,
           description:
-            'Updating may take a few seconds to take effect.\nPlease note that listening at a high volume for a long time may damage your hearing',
+            'Persistent volume updated.\nThis setting will stay across songs and restarts.\nPlease note that listening at a high volume for a long time may damage your hearing.',
```

Comment:
I'd suggest we never mention that it will persist. The user normally expects this to be the default. I just never got around to implement it in this bot.
As an example; Spotify or Youtube will "persist" accross videos or songs. Therefore, the user will expect the same here.